libtelemetry/src/androidTest/java/com/mapbox/android/telemetry/TelemetryServiceTest.java

```diff
@@ -174,8 +174,8 @@ private void stopService(Intent serviceIntent, TelemetryService[] telemetryServi
 
   private void waitUntilServiceIsDestroyed() throws InterruptedException {
     // Have to wait a bit until the system calls Service#onDestroy()
-    // Note that it only needs 10 milliseconds to clean the resources up
-    Thread.sleep(10);
+    // Note that it only needs 15 milliseconds to clean the resources up
+    Thread.sleep(15);
```

Comment:
How did we come up with the 15 milliseconds threshold?

Reply:
Intermittent failure of our `checksLocationReceiverIsDownWhenServiceStopped` test. Has been an issue for awhile. Pushed milliseconds to 15 to see if that would affect the test and it stopped failing. Probably can play with the value more, if desired.